### PR TITLE
Test emod3d update

### PR DESCRIPTION
**Pull request by jpa198**

### CODE REVIEW

File: scripts/test_emod3d.sh [hunk 1]
```diff
@@ -21,7 +21,17 @@ if [[ $? != 0 ]]; then
...
+fileCount=`ls -1|wc -l`
+
+#Check that the number of cores and number of rlog files is the same
+if [[ ! -z ${SLURM_CPUS_ON_NODE} ]] && [[ ${SLURM_CPUS_ON_NODE} != ${fileCount} ]]
+then
+    echo "Number of cores and number of log files mismatch"
+    exit 1
+fi
+
+rlog_count=$(expr 0)
```
[the remark this answers is not in the document]
Comment (jasonmotha) [reply scripts/test_emod3d.sh]: Also, isn't 0 the same as `expr 0`?

File: scripts/test_emod3d.sh [hunk 2]
```diff
@@ -32,13 +42,42 @@ do
...
+then
+    python3 -c "from qcore import timeseries; timeseries.LFSeis('../OutBin');" 2>/dev/null
+    seisIntegrity=$?
+    if [[ $seisIntegrity != 0 ]];
+    then
+        echo "At least one file failed the integrity check"
+    fi
+fi
+
+if [[ $rlog_check == 0 ]] && [[ $seisIntegrity == 0 ]];
```
Comment (jasonmotha) [inline scripts/test_emod3d.sh]: Put a comment here to explain what conditions cause emod3d to be not completed.

File: scripts/test_emod3d.sh [hunk 3]
```diff
@@ -32,13 +42,47 @@ do
...
+    if [[ $seisIntegrity != 0 ]];
+    then
+        echo "At least one file failed the integrity check"
+    fi
+fi
+
+# EMOD3D is not considered to be completed if:
+#   Any Rlog does not have IS FINISHED
+#   There is a seis file without an equally indexed xyts file or vice versa
+#   Any seis file is unable to be loaded by LFSeis
```
Comment (jasonmotha) [inline scripts/test_emod3d.sh]: How performant is the loading? Aka for large seis files does this take a while?

Comment (shot0829) [reply scripts/test_emod3d.sh]: for 400m AlpineF2K it usually take around 20~30 sec to load.

**Pull request merged**